### PR TITLE
Removes Roundstart Active Turfs

### PR DESCRIPTION
**Pull request by Robustin**
I got sick of MSO telling me to HEY LISTEN

The side effect is that lava/tunnels are less likely to directly "touch" certain ruins that have a lot of mineral walls around them but don't use an "unexplored" area type. 

If your lava ruin is surrounded by an "explored" Lava area, you can address this by replacing that area with a no-op area where appropriate.

Currently at 10 rounds with no roundstart AT but finding these things is down to luck now.

### CODE REVIEW

**Comment by Robustin** [comment]
Ok so far so good... except I figured out that the Ash Lizard ruin produces active turfs by default... gonna have to squeeze a mapedit into this.

**Comment by Robustin** [comment]
Looks like whoever did the swarner ruin already follow my advice in the OP and used a noop area. Unfortunately unless I figure out how to get rid of the air differential, I won't be able to stomp out ATs completely.

File: code/game/turfs/simulated/floor/plating/asteroid.dm [hunk 1]
```diff
@@ -211,6 +209,9 @@
 		if(istype(tunnel))
 			// Small chance to have forks in our tunnel; otherwise dig our tunnel.
 			if(i > 3 && prob(20))
+				if(!tunnel || isspaceturf(tunnel) || istype(tunnel.loc, /area/mine/explored) || (istype(tunnel.loc, /area/lavaland/surface/outdoors) && !istype(tunnel.loc, /area/lavaland/surface/outdoors/unexplored)))
```
Comment (AnturK) [inline code/game/turfs/simulated/floor/plating/asteroid.dm]: First two check on tunnel here are redundant you're doing it above anyway.

**Review by AnturK — DISMISSED** [review]

**Comment by Cyberboss** [comment]
_You look at mapdiffbot and sigh_

**Comment by Cyberboss** [comment]
@mapdiffBot

**Comment by Cyberboss** [comment]
@MapDiffBot

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**